tsdb/functions.go

```diff
@@ -1159,9 +1302,10 @@ func valueCompare(a, b interface{}) int {
 
 type PositionPoints []PositionPoint
```

Comment:
I know this isn't part of the PR, but any chance we could get a GoDoc on this type? It would help me review.